app/models/profile_field.rb

```diff
@@ -1,4 +1,8 @@
 class ProfileField < ApplicationRecord
+  before_create :generate_attribute_name
```

Comment:
This replaces the previous `attribute_name` method with a more sensible approach and pesists the name. Note that this name will never change again even if the field's label gets updated. This is so we can avoid rewriting every single user's stored profile for a wording change. @Ridhwana this means we're **not** going to expose `attribute_name` to admins.

Reply:
Sounds good!